client/src/services/timeline_animation.js

```diff
@@ -22,13 +21,14 @@ TimelineAnimation.prototype.runTimeline = function() {
   for (var i = 0; i < this.arrows.length; i++) {
     this.arrows[i].addEventListener('click', function(event){
       //determines whether to remove or add scrolling
+      console.log(event)
       const sign = (event.path[1].classList.contains("prev-arrow")) ? "" : "-";
       var modifier = 0;
       if(document.querySelector('#language-id')){
-        if(event.path[0].id  == 'back'){
```

Comment:
Issue with adding new element inside the button caused the path index to be pointing in the wrong place.